### PR TITLE
✨Add onClick property to existing buttons

### PR DESCRIPTION
**Pull request by fre-ben**
- Add onClick property to buttons
- Add second page to test redirecting on button click
- Add useRouter hook for redirecting

Deployment: https://sisu-pipelin-btnsprops-e5uj6ie.herokuapp.com/
I'll rename the test page and the redirecting function soon, it's just a quick test for now.
Part of issue #1

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
So much light in the early morning! So the day can start ☀

**Review by DNaunin — APPROVED** [review]
Good Job! Nice clean code. 🚀

File: components/button/ConfirmBtn.tsx [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import { ButtonProps } from "./BackBtn";
```
Comment (lmachens) [inline components/button/ConfirmBtn.tsx]: Good job reusing the typing here.
You could think about an isolated `button/types.ts` file.

File: pages/index.tsx [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 import Logo from "../components/logo/Logo";
 import styles from "../styles/Home.module.css";
 
 export default function Home() {
+  const router = useRouter();
+
+  const goToTest = (e) => {
+    e.preventDefault();
+    router.push("/test");
+  };
```
Comment (lmachens) [inline pages/index.tsx:14]: If the buttons are only used as link, better use `a`. 
If not, I would add `cursor: pointer` to your buttons to indicate, that you can click them.

**Review by lmachens — COMMENTED** [review]
gj 🚀

**Pull request merged**